tests/python/pants_test/backend/jvm/tasks/test_scalafmt.py

```diff
@@ -94,25 +94,22 @@ def task_type(cls):
     return ScalaFmtCheckFormat
 
   def test_scalafmt_fail_default_config(self):
-    self.set_options(skip=False)
```

Comment:
We default to `skip=False`, so these were unnecessary (unless somehow `pants.ini` leaks into TestBase tests? I don't think it does, right?)